cocos/renderer/gfx-metal/MTLPipelineState.mm

```diff
@@ -85,24 +86,50 @@ of this software and associated engine source code (the "Software"), a limited,
         CC_LOG_ERROR("CCMTLPipelineState: CC_NEW CCMTLGPUPipelineState failed.");
         return false;
     }
+    
+    if(_bindPoint == PipelineBindPoint::GRAPHICS) {
+        if (!createMTLRenderPipelineState()) {
+            return false;
+        }
+        // Application can run with wrong depth/stencil state.
+        if (!createMTLDepthStencilState()) return false;
```

Comment:
Please use the same code format for if statement. Can use
```
    if () xxx
   
    if ()
        xxx


    if () {
        xxx
    }
```
But please use the same format in a file.